crates/hdi/Cargo.toml

```
@@ -2,7 +2,7 @@
 name = "hdi"
 version = "0.4.0-beta-dev.13"
 description = "The HDI"
-license = "Apache-2.0"
+license-file = "LICENSE_CAL-1.0"
```

i think we can use `CAL-1.0` now. when we first started publishing it wasn't available yet but [it is on SPDX now](https://spdx.org/licenses/CAL-1.0.html).

```suggestion
license = "CAL-1.0"
```

---

Good point, updated!